### ISSUE TITLE
Remove "Next Word", Move and Improve "Stop Practice"

### ISSUE THREAD

**Issue by gsuthers** [ByunLab/staRt-ionic#73]
<Formerly "Touch Friendly 'Next Word' buttons>

The location of the "Next Word" buttons are inconvenient and workers harder to touch.

Could we please:
+ Remove the "Next Word/Syllable" buttons, causing the app to move to the next item after a rating is given.
+ Change the "Stop [Syllable/Word] Practice" to a button more like the reset button. Ideally an icon with a red square, but if necessary it can say 'stop trial' or 'stop session' for now.
+ Relocate "Stop [Syllable/Word] Practice"  button to the side of the wave, between the ratings and the reset.

**Comment by gsuthers** [comment]
Needs design from #75

**Comment by starakaj** [comment]
Am I waiting for some design here? Or is there some graphic I should be referring to?

**Comment by gsuthers** [comment]
@starakaj ya, this is part of the reason why I wanted to meet today. I have a temp graphic I can give you while we wait on Helen, but I think we should wait till she makes one- if it ends up holding us up I'll pass you a placeholder one.

**Comment by gsuthers** [comment]
@starakaj we've decided we want to remove the "Next Word" button and add a new stylized "Stop Trials" button to the right side of the wave, next to the feedback buttons as seen in the mockup below.
![quest3_mock](https://user-images.githubusercontent.com/11972125/30006051-d039062c-90bd-11e7-8e51-776a0f3ef110.png)

**I noticed that you put the 'reset wave' button below the feedback buttons and think that's great. It's better to have more commonly used buttons closer to the middle. I think the order should be (from top to bottom):**
Best rating
Middle rating
Lowest rating
Stop button
Reset wave

So we're just putting a stop button in between the ratings and the reset.
